torchgeo/datasets/millionaid.py

```diff
@@ -0,0 +1,397 @@
+        "apartment",
+        "apron",
+        "arable_land",
+        "bare_land",
+        "baseball_field",
+        "basketball_court",
+        "beach",
+        "bridge",
+        "cemetery",
+        "church",
+        "commercial_area",
+        "commercial_land",
+        "dam",
+        "desert",
+        "detached_house",
+        "dry_field",
+        "factory_area",
+        "forest",
+        "golf_course",
+        "grassland",
+        "greenhouse",
+        "ground_track_field",
+        "helipad",
+        "highway_area",
+        "ice_land",
+        "industrial_land",
+        "intersection",
+        "island",
+        "lake",
+        "leisure_land",
+        "meadow",
+        "mine",
+        "mining_area",
+        "mobile_home_park",
+        "oil_field",
+        "orchard",
+        "paddy_field",
+        "parking_lot",
+        "pier",
+        "port_area",
+        "power_station",
+        "public_service_land",
+        "quarry",
+        "railway",
+        "railway_area",
+        "religious_land",
+        "residential_land",
+        "river",
+        "road",
+        "rock_land",
+        "roundabout",
+        "runway",
+        "solar_power_plant",
+        "sparse_shrub_land",
+        "special_land",
+        "sports_land",
+        "stadium",
+        "storage_tank",
+        "substation",
+        "swimming_pool",
+        "tennis_court",
+        "terraced_field",
+        "train_station",
+        "transportation_land",
+        "unutilized_land",
+        "viaduct",
+        "wastewater_plant",
+        "water_area",
+        "wind_turbine",
+        "woodland",
+        "works",
+    ]
+
+    multi_class_categories = [
+        "apartment",
+        "apron",
+        "bare_land",
+        "baseball_field",
+        "bapsketball_court",
+        "beach",
+        "bridge",
+        "cemetery",
+        "church",
+        "commercial_area",
+        "dam",
+        "desert",
+        "detached_house",
+        "dry_field",
+        "forest",
+        "golf_course",
+        "greenhouse",
+        "ground_track_field",
+        "helipad",
+        "ice_land",
+        "intersection",
+        "island",
+        "lake",
+        "meadow",
+        "mine",
+        "mobile_home_park",
+        "oil_field",
+        "orchard",
+        "paddy_field",
+        "parking_lot",
+        "pier",
+        "quarry",
+        "railway",
+        "river",
+        "road",
+        "rock_land",
+        "roundabout",
+        "runway",
+        "solar_power_plant",
+        "sparse_shrub_land",
+        "stadium",
+        "storage_tank",
+        "substation",
+        "swimming_pool",
+        "tennis_court",
+        "terraced_field",
+        "train_station",
+        "viaduct",
+        "wastewater_plant",
+        "wind_turbine",
+        "works",
+    ]
+
+    md5s = {
+        "train": "1b40503cafa9b0601653ca36cd788852",
+        "test": "https://eastus1-mediap.svc.ms/transform/zip?cs=fFNQTw",
```

Comment:
This isn't an MD5 hash

Reply:
Yes, the test set is the much larger file, and I am not able to download it completely, to compute the md5.

Reply:
Can you try some kind of retry/continue on failure flag for wget/curl? Alternatively, you could email the authors and ask for the MD5 of the file (and warn them about the download issue).

Reply:
Sorry, what I meant to say is that my download issue is  that I do not have the disk space to download it completely.

Reply:
@calebrob6 does your VM have space?

Reply:
I'm downloading the test set to use for a side project and I'll report back with the hash.

Reply:
The hash for `test.zip` is `51a63ee3eeb1351889eacff349a983d8`